Segment/Classes/SEGContext.h

```diff
@@ -48,7 +48,6 @@ typedef NS_ENUM(NSInteger, SEGEventType) {
 @protocol SEGMutableContext;
 
 
-NS_SWIFT_NAME(Context)
```

Comment:
This could also be named something else, but this was the minimum change to resolve the naming conflict with SwiftUI